docsite/rst/playbooks_tests.rst

```diff
@@ -0,0 +1,165 @@
+Jinja2 tests
+============
+
+.. contents:: Topics
+
+
+Test in Jinja2 are a way of evaluating template expressions and returning True or False.
+Jinja2 ships with many of these. See `builtin tests`_ in the official Jinja2 template documentation.
+Tests are very similar to filters and are used mostly the same way, but they can also be used in list
+processing filters like C(map()) and C(select()) to choose items in the list.
+
+Just like filters, tests always execute on the ansible controller, NOT on the target of a task, they test local data.
+
+In addition to those, Ansible supplies a few more, also users can easily create their own.
```

Comment:
In addition to those Jinja2 tests, Ansible supplies a few more and users can easily create tests of their own.